webapp/channels/src/packages/mattermost-redux/src/selectors/entities/files.ts

```diff
@@ -46,6 +47,16 @@ export function makeGetFilesForPost(): (state: GlobalState, postId: string) => F
     );
 }
 
+export const getFilesForEditHistory: (state: GlobalState, editHistoryPost: Post) => FileInfo[] = createSelector(
```

Comment:
This should be a selector factory because it takes an argument

Reply:
Done.